types/stripe/index.d.ts

```diff
@@ -3969,6 +3971,69 @@ declare namespace Stripe {
             | 'incorporation_document';
     }
 
+    namespace fileLinks {
+        interface IFileLink extends IResourceObject {
+            /**
+             * Value is 'file_link'
+             */
+            object: 'file_link';
+            /**
```

Comment:
Can you separate all the comments and properties with a newline like so?

```
            /**
             * Value is 'file_link'
             */
            object: 'file_link';

            /**
             * Time at which the object was created. Measured in seconds since the Unix epoch.
             */
            created: number;
```

Reply:
done :) @pushplay 
thanks!